### ISSUE TITLE
Suggestion: Recycling burnt Food

### ISSUE THREAD

**Issue by K1ngPaul** [jasonrohrer/OneLifeData7#1163]
Problem: While accidentally burning food is a neat feature it kinda sucks that the resources are wasted and there is only one way to discard them. We should be able to use the remaining nutrients for something else like composted soil.

Solution: Allow us to add the **burnt items** ( burnt shrimp, burnt rabbit, burnt goose, burnt fish,) to **bowls** and then chop them down with a **sharp stone** or **knife** to create **burnt food mash**.
This burnt food mash could be added to **straw** to create **dry compost pile**

If the burnt goose is too big for the bowl tho, you could do the same thing but with flat stones. :)

**Comment by Shadie116** [comment]
I guess that is true, you smash up a chicken bone with a sledge harmer now you got bone mill

**Comment by Tarr-OHOL** [comment]
I frankly really like the idea as it allows a use for actually burning something. 

I don’t think it’s going to shake up the meta for composting because most people aren’t going to burning resources they could eat for compost. 

The only people really getting free compost material is probably ginger due to fishing producing mostly free fish after initial set up, and shrimp still require wool which requires water.

Rabbits require bait which is still water use, fed geese require some water (wild eggs are limited). Then of course we still have dry compost which will require water anyways.

**Comment by SoloAceMouse** [comment]
> Like... you go fishing or shrimping, and then get compost for less water than normal?

There are four burnt food items in-game along with their associated resource cost: [Disregarding the obvious kindling cost]
-**Burnt Fish** - Cost: 2.667 worms [approximate]
-**Burnt Goose** - Cost: 1 bowl of corn kernels
-**Burnt Rabbit** - Cost: 1/3 bowl of feed mash
-**Burnt Shrimp** - Cost: 1/4 small ball of yarn thread per shrimp [approximate]

-----

Of these four; goose, rabbit, and shrimp require appreciable "water-cost" to obtain, while fish are essentially free since worms are now a renewable resource.

All of the burnt foods would be cheaper in terms of water cost than traditional compost methods, however only fish would be a "freebie" in terms of water-cost.

The difference comes down to labor, all of the burnt foods require some effort to obtain that is greater than simply planting some seeds and harvesting a crop.

-----

tl;dr - Adding burnt foods as a compostable option would allow slightly cheaper water-cost options for compost, but would be offset by requiring a greater amount of time/effort to obtain.

**Comment by Rin0de0la0crypt** [comment]
Fish is the freebie with the worms being a direct by product with the worms, but if u instead make burnt fish edible maybe call it charred fish, and make it a yum different then cooked fish, this would make up for it but not make a food u can spam infinitely

**Comment by TwistedHBG** [comment]
If you're worried about burnt stuff being cheaper than using other materials just make it so you need to add the burnt food into a **bowl of water** to make the **burnt food mash**. That way you get a way to reuse the burnt food when someone burns it, but it won't be better than using the old methods.

Full disclaimer I haven't actually done the maths but at a glance I think this would be slightly more expensive than the old methods but still worth it when you end up with burnt stuff.

**Comment by Shadie116** [comment]
I think you should be able to smash up the newcomen in order to get compost because metal is good for plants

**Comment by jasonrohrer** [comment]
Good idea, Twisted!

**Comment by Shadie116** [comment]
thank you
![Screenshot 2024-01-03 031522](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/f6c4e6f2-9f36-47d1-a4ea-69d5a8c53049)
